frontend/views/components/i18n.vue

```diff
@@ -12,23 +13,21 @@ export default {
     tag: {
       type: String,
       default: 'span'
-    }
+    },
+    html: [String]
   },
-  methods: {
+  computed: {
     translatedText () {
-      const text = this.L(
-        this.$slots.default[0].text,
-        this.args || {},
-        { defaultValue: this.$slots.default[0].text }
-      )
+      const text = this.html || this.$slots.default[0].text
+      const translation = this.L(text, this.args || {}, { defaultValue: text })
```

Comment:
It would be nice to leave documented in this file how i18n can be used (as you explained in Slack) so people can remember it easily in the future.

Reply:
Good idea, but can we leave this for #607?